test/scripts/prune_namespaces.go

```diff
@@ -14,13 +14,20 @@
 
 package main
 
-import "github.com/kiegroup/kogito-cloud-operator/test/framework"
+import (
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/context"
+	"github.com/kiegroup/kogito-cloud-operator/test/framework"
+)
 
 func main() {
+	log := context.GetDefaultLogger()
```

Comment:
Please use the framework.GetMainLogger() instead as it is all done in the test package